code/style.css

```diff
@@ -1,5 +1,124 @@
-/* Here in the css file you'll write your styling. make sure to link the css file to the html file. If it's linked correctly you should see a light blue background. */
+@import url('https://fonts.googleapis.com/css2?family=Parisienne&display=swap');
 
 body {
-  background-color: lightblue;
+  background-color: rgba(243, 234, 204, 0.897);
+  font-family: "Parisienne", cursive;
+  color: black;
 }
+
+menu {
+  display: flex;
+  align-items: center;
+  justify-content: center;
+}
+
+menu a {
+  display: inline-block;
+  padding: 10px 15px;
+  background: rgba(243, 234, 204, 0.897);
+  color: black;
+  font-weight: bold;
+  text-decoration: none;
+  border-radius: 3px;
+  margin: 0 5px;
+}
+
+menu a:hover {
+  background:black;
+  color:rgba(243, 234, 204, 0.897);
+}
+
+@keyframes scale {
+  10% {transform: scale(2);} 
+  100% {transform: scale(1);}
+}
+
+.header {
+  width: auto;
+  height: 420px;
+  margin: 20px;
+  position: relative;
+}
+
+.header img {
+  width: 450px;
+  height: 350px;
+  border-radius: 50%;
+}
+
+img:hover{
+  animation-name: scale;
+  animation-duration: 10s;
+}
+
+.title {
+  text-align: center;
+  width: auto;
+  padding: 5px;
+  margin-left:100px;
+  margin-right: 100px;
```

Comment:
You could write this in one line like this, where the first value is top/bottom and the second one is left/right:
`margin: 0 100px;`